obal/data/playbooks/nightly/nightly.yaml

```diff
@@ -5,6 +5,8 @@
   gather_facts: no
   vars:
     build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args | default([]) }}"
+  roles:
+    - package_variables
```

Comment:
I needed to add this for `package_base_dir` to be defined. Not sure if there's a better way and if it's needed in this PR.